src/facsimilefunctor.cpp

```diff
@@ -184,7 +184,10 @@ FunctorCode SyncFromFacsimileFunctor::VisitStaff(Staff *staff)
         m_staffZones[staff] = zone;
 
         // The staff slope is going up. The y left position needs to be adjusted accordingly
-        if (zone->HasRotate() && zone->GetRotate() < 0) {
+        if (!zone->HasRotate()) {
+            zone->SetRotate(0);
```

Comment:
Why do not need it to actually set it to `0`? I think if you do this, then writing the file will have an explicit `0` value in the MEI, which should not be necessary.

Reply:
I see. `@rotate` is used in calculating closest staff. I will revert this commit and make other changes.